src/main.rs

```diff
@@ -207,7 +227,12 @@ fn run() -> Result<(), failure::Error> {
             None => None,
         };
 
-        commands::preview(project, user, method, body)?;
+        let watch = match matches.occurrences_of("watch") {
```

Comment:
match presence of not occurences .. because what if someone types `wrangler preview --watch --watch`